Formula/xray.rb

```diff
@@ -25,6 +25,11 @@ class Xray < Formula
     sha256 "b85355097f660164a65723223aaef0a62c671266d4a18927c5c12c32c036cc54"
   end
 
+  resource "example_config" do
+    url "https://raw.githubusercontent.com/XTLS/Xray-examples/main/VLESS-TCP-XTLS-WHATEVER/config_client/vless_tcp_xtls.json"
```

Comment:
This should come from a tagged release. As soon as the file is changed the build will fail.